index.md

```diff
@@ -12,35 +12,101 @@ First we set up & capture the content, then we render it in the scaffolding belo
 
 
 {% capture intro-left %}
-## ![Cockpit](/images/site/cockpit-logo.svg)
-{:.logo}
-
-<span aria-hidden="true" role="presentation">Cockpit</span> is a server manager that makes it easy to administer your GNU/Linux servers via a web browser.
-
+The [easy to use](#easy-to-use),
+[integrated](#integrated),
+[glanceable](#glanceable),
+and [open](#open-ended)
+web-based interface for your servers
 {% endcapture %}
 
 {% capture intro-right %}
 [![Docker screenshot]({{ site.baseurl }}/images/site/screenshot-docker.png)]({{ site.baseurl }}/images/site/screenshot-docker.png){:.screenshot.zoom}
 {% endcapture %}
 
 
+{% assign blurbs_highlight = 6 %}
+
 {% capture blurbs %}
 
-[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png){:.screenshot.zoom}
-### Easy to use
-Cockpit makes Linux discoverable, allowing sysadmins to easily perform tasks such as starting containers, storage administration, network configuration, inspecting logs and so on.
+## Easy to use
+
+### Discoverable
+Cockpit makes GNU/Linux discoverable. See your server in a web browser and perform system tasks with a mouse. It's easy to start containers, administer storage, configure networks, and inspect logs.
+
+[![Storage screenshot]({{ site.baseurl }}/images/site/screenshot-storage.png)]({{ site.baseurl}}/images/site/screenshot-storage.png)
+{:.screenshot.zoom}
+
+### Designed & tested
+Cockpit is designed with your goals in mind. We also routinely test Cockpit with usability studies to make it work the way you'd expect. As a result, Cockpit gets easier to use all the time.
+
+### Team-friendly
+Cockpit is friendly enough for those new to Linux and useful for seasoned admins too.
+
+### Packages included
+[Installing and running Cockpit]({{ site.baseurl }}/running.html) is simple. It's already included in most of the major distributions.
 
 ---
 
-[![Network screenshot]({{ site.baseurl }}/images/site/screenshot-network.png)]({{ site.baseurl }}/images/site/screenshot-network.png){:.screenshot.zoom}
-### No interference
-Jumping between the terminal and the web tool is no problem. A service started via Cockpit can be stopped via the terminal. Likewise, if an error occurs in the terminal, it can be seen in the Cockpit journal interface.
+## Integrated
+
+### Plays well with others
+Jumping between the terminal and the web tool is no problem.
```

Comment:
I don't particularly like the "no problem". A bit hard to express why, maybe it's not positive/encouraging enough. How about "You can jump between the terminal and the web interface at any time."?

Reply:
Agreed. I'll update this.

Reply:
(Note: This is some of the text that was recycled from the front page of the current site. It's there on Cockpit-project.org right now. But this is definitely a good time to revise it.)